adafruit_is31fl3731/__init__.py

```diff
@@ -239,11 +253,17 @@ def frame(self, frame=None, show=True):
             self._register(_CONFIG_BANK, _FRAME_REGISTER, frame)
         return None
 
-    def audio_sync(self, value=None):
+    def audio_sync(self, value: int = None) -> int:
         """Set the audio sync feature register"""
         return self._register(_CONFIG_BANK, _AUDIOSYNC_REGISTER, value)
 
-    def audio_play(self, sample_rate, audio_gain=0, agc_enable=False, agc_fast=False):
+    def audio_play(
+        self,
+        sample_rate: int = 0,
```

Comment:
I think it's best to leave `sample_rate` without a default so that it remains a required argument as it was written originally.